### DISCUSSION TITLE
Notice lifecycle (Publishing status in preview)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#606] [Q&A] [answered]
In the preview environment we see that the Forms we send go directly from Submitted to Published status. Does preview never go through the Publishing state? In the production environment, will they go through the Publishing state?

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/7a716d70-ff85-495c-9527-006041765943)

Thanks in advance,
Alex

**Comment by ferrakaop** [comment, marked as answer]
All (valid) notices go through the Publishing status.  In the Preview environment, the notice is only in Publishing status for a few seconds because there is a mock process to imitate publishing on TED.  In Production, the notice will be in Publishing status between the daily afternoon export and publication on TED the next morning (working days).

There are more details at https://docs.ted.europa.eu/home/eforms/preview/index.html#_known_enotices2_api_issues and we'll add these clarifications.

Please note that the Publishing status is still yet not visible for eNotices2 web or API users.  It's planned for a future release.
